ckanext/canada/commands.py

```diff
@@ -313,7 +316,30 @@ def _changed_packages_since(self, registry, since_time):
         for result in data:
             package_id = result['data']['package']['id']
             try:
-                packages.append(json.dumps(registry.action.package_show(id=package_id)))
+                source_package = registry.action.package_show(id=package_id)
+                for source_resource in source_package['resources']:
+                    # check if resource exists in datastore
+                    source_table = registry.call_action('datastore_search',
+                                                        {'id': source_resource['id'], 'limit': 0})
+                    if source_table:
+                        from ckanext.datastore.helpers import datastore_dictionary
+                        import simplejson
+
+                        # fetch all records
+                        source_records = registry.call_action('datastore_search',
+                                                              {'id': source_resource['id'],
+                                                               'limit': source_table['total']})
+                        for record in source_records['records']:
+                            record.pop('_id', None)
```

Comment:
you can pass fields to datastore_search to exclude the _id column from the response instead of doing the work afterwards